pkg/ansible/runner/runner.go

```diff
@@ -51,6 +53,7 @@ type watch struct {
 	Playbook        string     `yaml:"playbook"`
 	Role            string     `yaml:"role"`
 	ReconcilePeriod string     `yaml:"reconcilePeriod"`
+	ManagedStatus   string     `yaml:"managedStatus"`
```

Comment:
`*bool` instead of `string` I think. Nit, but the yaml value should be `manageStatus` I think.